src/CMakeLists.txt

```diff
@@ -117,6 +117,47 @@ install( TARGETS gauxc ${GAUXC_EXPORT_DEPENDENCIES}
 
 set_target_properties( gauxc PROPERTIES EXPORT_NAME gauxc )
 
+set(export_properties
+  # currently configurable properties 
+  "GAUXC_ENABLE_TESTS"
```

Comment:
I cant think of a good reason this should be exported, tests are meant to be encapsulated.

Reply:
Yeah, that does make sense. Fixed!